docs/welcome/introduction.md

```diff
@@ -0,0 +1,29 @@
+---
+tags: [00. Welcome]
+---
+
+# Intro
+
+Studio is our next generation app for API design, modeling, and technical writing. 
+
+> A primary goal of Studio is to enrich, not replace, your existing workflows. It works offline, with folders and files on your computer, just like your favorite IDEs.
+
+Here is some of what it can do:
+
+- **OpenAPI v2 AND v3:** Form based design mode means you don't need to be an OpenAPI expert to get started. Studio also supports OpenAPI autocomplete in write mode, a read mode to visualize HTTP operations and models, mocking, and linting for OpenAPI v2 and v3.
+- **Standalone JSON Schema Modeling:** Studio encourages you to split your models into separate files, and then makes it easy to create `$refs` between them.
+- **Stoplight Flavored Markdown:** SMD is an optional, lightweight extension to regular markdown. It enables a few advanced features such as tabs and callouts.
```

Comment:
This sounds scary and would worry me if i was about to use it. Its not really a special flavour we just have some conventions around code block names and certain meanings, right? It's still the same syntax, just with extra meaning?

Also can we mention if we are Commonmark based, or which flavour we are based on? 

Markdown is a mess: https://phil.tech/markdown/2014/11/30/state-of-markdown/

Reply:
@marbemac review